product-mini/platforms/posix/main.c

```diff
@@ -378,6 +394,8 @@ main(int argc, char *argv[])
     int instance_port = 0;
 #endif
 
+    memset(&init_args, 0, sizeof(RuntimeInitArgs));
```

Comment:
Define running_mnode = 0 at the begining of the function and don't move the line here.